src/main.cpp

```diff
@@ -125,9 +125,8 @@ QCoreApplication* createApplication(int& argc, char* argv[], unsigned int& actio
         // before constructing QGuiApplication."
         QApplication::setAttribute(Qt::AA_ShareOpenGLContexts);
 #elif defined(Q_OS_WIN32)
-        // Force OpenGL use as we use some functions that aren't provided by
-        // Qt's OpenGL layer on Windows (QOpenGLFunctions)
-        QApplication::setAttribute(Qt::AA_UseDesktopOpenGL);
+        // Force ANGLE use - some nvidia cards on windows tend to 'freeze'
+        QApplication::setAttribute(Qt::AA_UseOpenGLES);
```

Comment:
Do we HAVE to force a particular system on Windows - this prevents the two alternatives that other users may want/need/prefer - see https://doc.qt.io/qt-5/windows-requirements.html#dynamically-loading-graphics-drivers for the gory details but allowing the user to use `QT_OPENGL` to set their requirements seems like a less draconian methodology - Qt even has a built in (but overridable) system to choose the least buggy option for a particular graphics card if it is NOT forced like this...

Reply:
From my research on this it looks like no option is ideal:

```
(forced angle) Qt::AA_UseOpenGLES - no hang on neither gtx1070 nor 1080, but nothing is shown

(forced opengl) Qt::AA_UseDesktopOpenGL - hang on both cards

no flag - 1080 runs fine, gtx1070 hangs
```

In the end I think we'll need a failsafe option - allow the player to choose, and also auto-unselect the option if it failed to work and hung/crashed Mudlet due to a bad video driver.

Reply:
Um, did you try the third forcing option: `Qt::AA_UseSoftwareOpenGL`?

Reply:
> Um, did you try the third forcing option: Qt::AA_UseSoftwareOpenGL?

No, as that would be bad on performance.

> you (or the end-user) can control it by setting the environment variable

We have people in #help saying they've met people who are terrified of copy/pasting code - our population does not know what an environment variable is nor do they want to :joy: